src/index.html

```diff
@@ -7,6 +7,316 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header__logo">
+        <img
+          src="images/logo.png"
+          alt="logo"
+        >
+      </a>
+
+      <nav>
+        <ul class="nav">
+          <li class="nav__li">
+            <a
+              class="nav__link is-active" href="#"
+            >
+            Apple
+            </a>
+          </li>
+
+          <li class="nav__li">
+            <a class="nav__link" href="#">
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__li">
+            <a class="nav__link" href="#">
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__li">
+            <a
+              class="nav__link laptops"
```

Comment:
The 'class="nav__link laptops"' should be 'class="nav__link nav__link--laptops"' for proper BEM naming convention.